assets/src/components/amp-support/validated-urls.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ListItems } from '../list-items';
+
+/**
+ * To render validated urls information on site support page.
+ *
+ * @param {Object} props               Component props.
+ * @param {Array}  props.validatedUrls Error data.
+ * @return {JSX.Element|null} HTML markup for validated urls data.
+ */
+export function ValidatedUrls( { validatedUrls } ) {
+	if ( ! Array.isArray( validatedUrls ) ) {
+		return null;
+	}
+
+	const urls = validatedUrls.map( ( item ) => item.url ? item.url : null );
+	const items = urls.map( ( url ) => {
+		return {
+			url,
+		};
+	} );
```

Comment:
I think this could be done in one step (not tested):

```suggestion
	const items = validatedUrls.map( ( item ) => ( {
		url: item.url ?? null,
	} ) );
```

Reply:
This will also work.